app/Exceptions/LettersException.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * LettersException
+ *
+ * PHP version 8
+ *
+ * @category Exceptions
+ * @package  Exceptions\Address
+ * @author   Angel García <angelgarciaweb@gmail.com>
+ * @license  MIT https://opensource.org/licenses/MIT
+ * @link     https://github.com/jvngarcia/devathon-8-backend.git
+ */
+
+namespace App\Exceptions;
+
+use Exception;
+
+/**
+ * Letters exception
+ *
+ * @category Exceptions
+ * @package  Exceptions\Letters
+ * @author   Angel García <angelgarciaweb@gmail.com>
+ * @license  MIT https://opensource.org/licenses/MIT
+ * @link     https://github.com/jvngarcia/devathon-8-backend.git
+ */
+class LettersException extends Exception
+{
+    //
```

Comment:
You left this exception empty, is it no longer needed or did you forget to fill it?